google/devtools/clouddebugger/clouddebugger.yaml

```diff
@@ -0,0 +1,38 @@
+type: google.api.Service
+
+config_version: 2 # (assumed)
```

Comment:
We shouldn't make this assumption. The config version is available from the original internal service config.